src/overlays/misc/ovl_kaleido_scope/z_kaleido_collect.c

```diff
@@ -24,20 +24,21 @@ void KaleidoScope_DrawQuestStatus(PlayState* play, GraphicsContext* gfxCtx) {
 
     // Shining medallions
     static s16 sMedallionsEnvColors[6 + 6][3] = {
-        // Target env color when sMedEnvShineState == 0
-        { 0, 0, 0 },
-        { 0, 0, 0 },
-        { 0, 0, 0 },
-        { 0, 0, 0 },
-        { 0, 0, 0 },
-        { 0, 0, 0 },
-        // Target env color when sMedEnvShineState == 2
-        { 0, 60, 0 },
-        { 90, 0, 0 },
-        { 0, 40, 110 },
-        { 80, 40, 0 },
-        { 70, 0, 90 },
-        { 90, 90, 0 },
+        // Target env color when sMedallionsEnvShineState == 0
+        { 0, 0, 0 }, // QUEST_MEDALLION_FOREST
```

Comment:
unrelated to your changes here but will there eventually be an enum for "shine state"? I dont like having c code in the comment like this lol

Reply:
I don't plan to have an enum no

Reply:
oh. well, "state" with just raw numbers is usually pretty hard to read, implies to me that these states should have names. but guess this should be discussed outside of this pr